app/src/main/java/com/eventyay/organizer/core/auth/start/StartFragment.java

```diff
@@ -63,7 +66,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     @Override
     public void onStart() {
         super.onStart();
-
+        startFragmentViewModel.getWarningDialog().observe(this, prefernceKey -> {
+            if (!prefernceKey) {
```

Comment:
No logic should be placed here. Simply send the callback only if preference key is not set

Reply:
Okay Sir